sdk/agrifood/agrifood-farming-rest/samples-dev/listFarmers.ts

```diff
@@ -21,14 +21,16 @@ async function main() {
   const response = await farming.path("/farmers").get();
 
   if (response.status !== "200") {
-    throw response.body.error || new Error(`Unexpected status code ${response.status}`);
+    const error = <ErrorResponseOutput>response.body;
```

Comment:
Same here if you change line 23 to

```ts
if (isUnexpected(response)) {
```

You don't need to cast. This comment applies to all casts in this PR, we shouldn't need any explicit cast

Reply:
Done